addons/account/tests/test_account_supplier_invoice.py

```diff
@@ -98,7 +98,7 @@ def test_supplier_invoice2(self):
         # Keep in mind that tax amount can be changed by the user at any time before validating (based on the invoice and tax laws applicable)
         invoice_tax = invoice.tax_line_ids.sorted(key=lambda r: r.sequence)
         self.assertEquals(invoice_tax.mapped('amount'), [50.0, 550.0, 220.0])
-        self.assertEquals(invoice_tax.mapped('base'), [500.0, 1100.0, 1100.0])
+        self.assertEquals(invoice_tax.mapped('base'), [500.0, 550.0, 1100.0])
```

Comment:
imho 550 base is tax excluded so you can't just mix it with tax included in price